docs/software-platforms/aws/starting-development-on-lsfr.md

```diff
@@ -0,0 +1,78 @@
+# Getting started with development on Lightsail for Research
+
+Once you have [created a virtual computer](creating-vcs.md), [attached a disk](working-with-volumes.md), and [accessed the operating system](creating-vcs.md#how-to-access-a-virtual-computer) you are ready to start developing analyses on Lightsail for Research.
```

Comment:
It might be helpful to just list the general steps that you go through later in the doc here. So people can see an overview of what they need to do before they can actually do any analysis.